Source/Adobe.Target.Client/Model/TargetDeliveryResponse.cs

```diff
@@ -84,6 +84,22 @@ public sealed class TargetDeliveryResponse
             }
         }
 
```

Comment:
Why are we expanding the public API?

Reply:
Ideally we should modify `VisitorState` itself to return a `string`, as there's a really low chance it will be used as a dictionary. However, due to us already having it exposed in the Java SDK for a longer period of time as a map, we can't make this breaking change now just in the .NET SDK. So, it's better to expand the API with a `VisitorStateJson` method.
Also, it's better we serialize the visitor state inside the SDK, as it's not as simple as serializing it as-is and we do need a custom serializer for it.

Reply:
Renamed to `VisitorStateAsJson` as agreed